packages/cfd/src/Containers/mt5-create-password/mt5-create-password.scss

```diff
@@ -0,0 +1,226 @@
+.dc-modal__container_cfd-password-modal,
+.dc-mobile-dialog__cfd-password-modal {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+
+    form {
+        flex-grow: 1;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        justify-content: space-between;
+        padding: 0 2.4rem 1.2rem;
+
+        @include mobile-or-tablet-screen {
+            height: 100%;
+        }
+    }
+    &__description {
+        text-transform: none;
+        margin-top: 1rem;
+        top: 2rem;
+        position: relative;
+        padding: 1.3rem;
+    }
+    &__hint {
+        max-width: 40rem;
+    }
+    &__body {
+        flex-grow: 1;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: flex-start;
+    }
+    .dc-password-meter {
+        &__bg {
+            background: var(--general-section-2);
+            @include mobile-or-tablet-screen {
+                background: var(--fill-disabled);
+            }
+        }
+    }
+    & .dc-password-meter__container {
+        flex-grow: 1;
+        margin: auto;
+
+        @include mobile-or-tablet-screen {
+            width: 100%;
+            max-width: 100%;
+        }
+    }
+    & .mt5-password-field {
+        margin-bottom: 1rem;
+        width: 80%;
+    }
+    .input-element {
+        width: 100%;
+        margin-bottom: 1rem;
+    }
+    @include mobile-or-tablet-screen {
+        .cfd-password-modal__content {
+            overflow-y: auto;
+            overflow-x: hidden;
+            height: 100%;
+            width: 100%;
+            max-height: calc(100vh - 6.4rem);
+        }
+    }
+    @include tablet-up {
+        .cfd-password-modal__content {
+            display: flex;
+            flex-direction: column;
+            width: 100%;
+            margin-bottom: 2rem;
+        }
+    }
+}
+
+.cfd-password-modal {
+    &__warning {
+        padding: 2.5rem;
+        max-width: 50rem;
+        align-self: center;
+        &-text {
+            border-radius: 0.8rem;
+            padding: 0.8rem;
+            background-color: var(--status-warning);
+        }
+    }
+    &__message {
+        max-width: 32rem;
+        margin: auto;
+        line-height: 1.43;
```

Comment:
why no unit?

Reply:
have removed __message as its not being used by MT5Creaste password modal

Reply:
is it done?